ui/app/public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="stylesheet" href="//static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css" />
```

Comment:
Out of interest, why would we use the protocol-relative url, rather than `https`?
This is generally not a good thing.

Reply:
is this needed by rsjf ? presume because it's styling things to look like fluent, rather than using the fluent components?

Reply:
Yes, just following their documentation - I believe this is to use the fabric grid layout. I think we leave for now and investigate whether this can be removed/worked-around during refinements

Reply:
I would still suggest you go back and remove the protocol-relative stuff.
We never want anything other than https for dependencies